Dockerfile

```diff
@@ -2,12 +2,12 @@ FROM ubuntu:18.04
 
 RUN apt-get update && apt-get install -y curl git
 ENV PATH /root/.rbenv/bin:/root/.rbenv/shims:$PATH
-RUN curl -fsSL https://github.com/rbenv/rbenv-installer/raw/master/bin/rbenv-installer | bash
+RUN curl -fsSL https://github.com/rbenv/rbenv-installer/raw/HEAD/bin/rbenv-installer | bash
```

Comment:
This seems like an odd place to get the data from.  You might want to change this to main instead of HEAD, as that will be more reliable in terms of knowing which branch you're working with.

Reply:
Their URL will have changed from:
https://github.com/rbenv/rbenv-installer/raw/master/bin/rbenv-installer

to:
https://github.com/rbenv/rbenv-installer/raw/main/bin/rbenv-installer